raglweb/__init__.py

```diff
@@ -284,6 +284,18 @@ def player(profile_id):
 def info():
     return render_template('info.html', start_time=_cfg['start_time'])
 
+def _make_division_prefix(division_title):
+    """Convert the human readable division title into a prefix for use in the filename."""
+    words = division_title.upper().split(' ')
+    prefix = 'RAGL-S10-' + words[0]
```

Comment:
S10 has the same problem: i'm afraid we'll forget it: could you add a `_cfg['season']`?